must-gather/collection-scripts/gather

```diff
@@ -10,11 +10,11 @@ gather_namespaced_resources ${BASE_COLLECTION_PATH} "${SINCE_TIME}"
 gather_clusterscoped_resources ${BASE_COLLECTION_PATH} "${SINCE_TIME}"
 gather_noobaa_resources ${BASE_COLLECTION_PATH} "${SINCE_TIME}"
 
-
-
```

Comment:
lines removed any specific reason

Reply:
Unnecessary extra lines, I guess one line is enough to leave.

Reply:
cool